packages/eslint-plugin/src/rules/prefer-readonly.ts

```diff
@@ -300,6 +300,7 @@ class ClassScope {
         node.name.kind === ts.SyntaxKind.PrivateIdentifier
       ) ||
       tsutils.isModifierFlagSet(node, ts.ModifierFlags.Readonly) ||
+      tsutils.isModifierFlagSet(node, ts.ModifierFlags.Accessor) ||
```

Comment:
[Refactor] Whoops, sorry, missed this till now ~

```suggestion
      tsutils.isModifierFlagSet(node, ts.ModifierFlags.Accessor | ts.ModifierFlags.Readonly)
```

..since we're releasing in the next hour I'll just go ahead and apply.